### PR TITLE
Calculate stats on save

### PR DESCRIPTION
**Pull request by sethvargo**
- Fixes GH-200
- Closes GH-283

**Release Note**

```release-note
Calculate app and user statistics on save
```

### CODE REVIEW

File: cmd/server/assets/users/show.html [hunk 1]
```diff
@@ -77,6 +81,32 @@ <h1>{{$user.Name}}</h1>
   </main>
 
   {{template "scripts" .}}
+
+  {{if $stats}}
+  <script src="https://www.gstatic.com/charts/loader.js"></script>
```
Comment (mikehelmick) [inline cmd/server/assets/users/show.html:86]: push this JS out to a sub-template?

Comment (sethvargo) [reply cmd/server/assets/users/show.html:86]: The only JS that's "common" is the URL for loader.js, and it didn't seem worth adding that layer of indirection.

Comment (mikehelmick) [reply cmd/server/assets/users/show.html:86]: scrolling back and forth, they looked more similar to me :)

**Pull request merged**